lib/person.rb

```diff
@@ -0,0 +1,36 @@
+require 'date'
+require 'pry'
+
+class Person
+  attr_accessor :portfolio
+
+  def initialize
+    @portfolio = []
+  end
+
+  def rent(library, args={})
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for rent is too high. [24.27/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [15/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)